cli/cmd/inject.go

```diff
@@ -319,7 +325,11 @@ func (resourceTransformerInject) generateReport(reports []inject.Report, output
 
 	for _, r := range reports {
 		if b, _ := r.Injectable(); b {
-			output.Write([]byte(fmt.Sprintf("%s \"%s\" injected\n", r.Kind, r.Name)))
+			verb := "injected"
+			if r.Kind == k8s.Service {
```

Comment:
I think using the "injected" verb is fine for services too, even though we only add annotations.  For example, we already use the verb "injected" for namespaces (which also only get an annotation)